openpype/pipeline/actions.py

```diff
@@ -100,6 +102,11 @@ def process(self, containers):
         """
         return True
 
+    @classmethod
+    def filepath_from_context(cls, context):
+        return get_representation_path_from_context(context)
+
+
```

Comment:
Isn't this one empty line too many?
```suggestion
```

Reply:
> Isn't this one empty line too many?

shouldn't be, next function isn't part of the class `InventoryAction` so it should be separated by two lines

Reply:
I thought github was showing me three lines somehow :/ sorry,